File: 2D_amber/Assets/Scripts/CSharp/Utility/GoToScene.cs
using UnityEngine;
using System.Collections;

public class GoToScene : MonoBehaviour {

	public string sceneName = "" ;
	public int sceneId = 0 ;

	public void GoTo(){
  		if(sceneName == ""){
    		Game.screen().FadeAndGo(sceneId) ;
  		}else{
    		Game.screen().FadeAndGo(sceneName) ;
  		}
	}
}

File: 2D_amber/Assets/Scripts/JS/Utility/GoToScene.js
#pragma strict
import UnityEngine.SceneManagement ;

var sceneName : String = "" ;
var sceneId : int = 0 ;

function GoTo(){
  if(sceneName == ""){
    Game.screen().FadeAndGo(sceneId) ;
  }else{
    Game.screen().FadeAndGo(sceneName) ;
  }
}
